src/main/java/uk/gov/hmcts/divorce/divorcecase/model/FinalOrder.java

```diff
@@ -173,6 +186,119 @@ public enum Granted implements HasLabel {
     )
     private TranslatedToLanguage applicant2FinalOrderLateExplanationTranslatedTo;
 
+    @CCD(
+        label = "Explain why you need to apply for the final order",
+        access = {Applicant2Access.class},
+        typeOverride = TextArea
+    )
+    private String applicant2SolFinalOrderWhyNeedToApply;
+
+    @CCD(
+        label = "Respondent solicitor applied for final order?",
+        access = {DefaultAccess.class}
+    )
+    @Builder.Default
+    private YesOrNo applicant2SolAppliedForFinalOrder = NO;
+
+    @CCD(
+        label = "Date respondent solicitor applied for final order",
+        access = {DefaultAccess.class}
+    )
+    @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS")
+    private LocalDateTime dateApplicant2SolAppliedForFinalOrder;
+
+    @CCD(
+        label = "Respondent solicitor responsible for final order application",
+        access = {DefaultAccess.class}
+    )
+    private String applicant2SolResponsibleForFinalOrder;
+
+    @CCD(
+        label = "Here are your order details",
+        access = {DefaultAccess.class}
+    )
+    private OrderSummary applicant2SolFinalOrderFeeOrderSummary;
+
+    @CCD(
+        label = "Respondent solicitor final order fee (in pounds)",
+        access = {DefaultAccess.class}
+    )
+    private String applicant2SolFinalOrderFeeInPounds;
+
+    @CCD(
+        label = "How is payment being made?",
+        typeOverride = FixedRadioList,
+        typeParameterOverride = "SolicitorPaymentMethod",
+        access = {DefaultAccess.class}
+    )
+    private SolicitorPaymentMethod applicant2SolPaymentHowToPay;
+
+    @JsonIgnore
+    public boolean isSolicitorPaymentMethodPba() {
+        return FEE_PAY_BY_ACCOUNT.equals(this.getApplicant2SolPaymentHowToPay());
+    }
+
+    @CCD(
+        label = "Account number",
+        access = {DefaultAccess.class}
+    )
+    private DynamicList pbaNumbers;
```

Comment:
Question: is it possible to re-use the Application pbaNumbers list instead of a new field? As we're not updating it, but rather querying and loading it for use?